ufs2arco/regrid_mom6.py

```diff
@@ -0,0 +1,302 @@
+import os
+import yaml
+import xarray as xr
+import numpy as np
+import xesmf as xe
+from .gaussian_grid import gaussian_latitudes
+
+
+class RegridMOM6:
+    """
+    Regrid ocean dataset that is on a tripolar grid to a different grid (primarily Gaussian grid).
+
+    Required fields in config:
+        rotation_file (str): path to file containing rotation fields "sin_rot" and "cos_rot"

```

Comment:
Actually I suggest we make this optional since we may want to only regrid tracer fields.
```suggestion
```